client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -87,27 +83,19 @@ const ProfessionalEmailCard = ( {
 	source,
 }: EmailProvidersStackedCardProps ): ReactElement => {
 	const hasCartDomain = Boolean( cartDomainName );
-	const currencyCode = useSelector( getCurrentUserCurrencyCode );
 	const selectedSite = useSelector( getSelectedSite );
 	const domains = useSelector( ( state ) => getDomainsBySiteId( state, selectedSite?.ID ) );
 	const domain = getSelectedDomain( {
 		domains,
 		selectedDomainName: selectedDomainName,
-	} ) as ResponseDomain;
+	} );
 
 	const cartKey = useCartKey();
 	const shoppingCartManager = useShoppingCart( cartKey );
 
 	const professionalEmail: ProviderCard = { ...professionalEmailCardInformation };
 	professionalEmail.detailsExpanded = detailsExpanded;
 
-	const isEligibleForFreeTrial = hasCartDomain || isDomainEligibleForTitanFreeTrial( domain );
```

Comment:
Note I removed `hasCartDomain` from the new component, since we don't use that prop right now. We can add it back if we ever need it again.